packages/vip-go/services/logService.js

```diff
@@ -24,10 +13,17 @@ const defaultService = {
  */
 const getService = (namespace) => {
   const env = process.env.NODE_ENV;
-  let service;
+  let service = defaultService;
```

Comment:
check that this uses core's logging as a fallback (and maybe write a test to that effect as well)